JustFakeIt.Tests/AcceptanceTests/FakeServerTimingScenarios.cs

```diff
@@ -43,14 +39,26 @@ public class FakeServerTimingScenarios
                 fakeServer.Expect.Get("/some-path").Returns(expectedResult).RespondsIn(TimeSpan.FromSeconds(5)).WithHttpStatus(HttpStatusCode.OK);
                 fakeServer.Start();
 
-                var stopwatch = Stopwatch.StartNew();
+                // Do not use stopwatch:
+                var elapsedTime = await CallFakeServerWithAccurateUtcTimer(fakeServer);
+
+                elapsedTime.Should().BeGreaterOrEqualTo(expectedResponseTime);
+            }
+        }
+
+        private static async Task<TimeSpan> CallFakeServerWithAccurateUtcTimer(FakeServer fakeServer)
+        {
+            // The Stopwatch class uses the Windows "performance counter". 
+            // I have often read that on some systems it returns inaccurate data. 
+            // This appears to happen with older hardware and/or older operating system versions.
+            // Stack overflow: http://stackoverflow.com/questions/36725825/c-sharp-await-task-delay1000-only-takes-640ms-to-return
```

Comment:
I think the link alone is a will suffice, you don't need the comments justifying the change IMO